src/test/java/com/glencoesoftware/bioformats2raw/test/ZarrTest.java

```diff
@@ -1222,11 +1222,11 @@ public void testMinSizeOption() throws Exception {
     List<Map<String, Object>> datasets =
               (List<Map<String, Object>>) multiscale.get("datasets");
 
-    String last_path = (String) datasets.get(datasets.size()  - 1).get("path");
-    ZarrArray last_array = z.openArray(last_path);
+    String lastPath = (String) datasets.get(datasets.size()  - 1).get("path");
+    ZarrArray lastArray = z.openArray(lastPath);
 
-    assert last_array.getShape()[0] < 32;
-    assert last_array.getShape()[1] < 32;
+    assert lastArray.getShape()[0] < 32;
+    assert lastArray.getShape()[1] < 32;
```

Comment:
Should this be checking array elements 3 and 4 (Y and X)? I was expecting this test to fail, since the default fake image size is 512x512, which means the smallest resolution should be 32x32.

Reply:
:+1: updated in https://github.com/glencoesoftware/bioformats2raw/pull/114/commits/a8989140de62595eedacea5631c0eca46d87c2d9 to use `assertEqualArrays` on the return of `getShape`